examples/03_promise/src/components/Counter.tsx

```diff
@@ -26,9 +26,12 @@ const Message = ({
 }: {
   count: number;
   delayedMessage: Promise<string>;
-}) => (
-  <ul>
-    <li>count: {count}</li>
-    <li>delayedMessage: {delayedMessage as any}</li>
-  </ul>
-);
+}) => {
+  const message = use(delayedMessage);
+  return (
+    <ul>
+      <li>count: {count}</li>
+      <li>delayedMessage: {message}</li>
```

Comment:
I would like to merge this with something like this:
```suggestion
     {/* We want show the usage without `use` but it causes a hydration error. https://github.com/dai-shi/waku/issues/202 */}
      <li>delayedMessage: {use(delayedMessage)}</li>
```